lib/src/testing/test-runner.spec.ts

```diff
@@ -176,7 +176,7 @@ describe("test runner", () => {
     expect(result).toBe(true);
   });
 
-  test("include draft", async () => {
+  test("draft endpoint included", async () => {
     const contract = parseAndCleanse(
       `${testExamplesBasePath}/draft-endpoint.ts`
```

Comment:
Is there a reason why the contract used isn't the same as for `draft endpoint excluded`?